stock_quant_package_dimension/models/stock_quant_package.py

```diff
@@ -6,18 +6,24 @@
 class StockQuantPackage(models.Model):
     _inherit = "stock.quant.package"
 
-    pack_weight = fields.Float("Weight")
-    pack_length = fields.Integer("Length", help="length")
-    width = fields.Integer("Width", help="width")
-    height = fields.Integer("Height", help="height")
+    pack_weight = fields.Float("Pack Weight")
```

Comment:
You can remove string attribute.

```suggestion
    pack_weight = fields.Float()
```